integration/client/images/images_test.go

```diff
@@ -410,3 +413,84 @@ func TestImageBadTag(t *testing.T) {
 	assert.Equal(t, "repository can only contain the characters `abcdefghijklmnopqrstuvwxyz0123456789_-./`: foo@@", err.Error())
 
 }
+
+func TestImageListBuildTwoImagesAssertMultiClientLists(t *testing.T) {
```

Comment:
Is there something that is being tested here that requires this be an integration test? Can this be a unit test instead?